crates/bitwarden-crypto/src/lib.rs

```diff
@@ -80,8 +80,7 @@ mod util;
 pub use util::{generate_random_alphanumeric, generate_random_bytes, pbkdf2};
 mod wordlist;
 pub use wordlist::EFF_LONG_WORD_LIST;
-mod allocator;
-pub use allocator::ZeroizingAllocator;
+pub use zeroizing_alloc::ZeroAlloc as ZeroizingAllocator;
```

Comment:
Should we rename ours to `ZeroAlloc`?

Reply:
I kept the old name for backwards compatibility, probably not really needed but it was simple enough so I didn't see a reason not to.

We can use `ZeroAlloc` and ditch the old naming, if we don't consider that useful.

Reply:
I guess there's no harm keeping the current name.